### ISSUE TITLE
Suche nach Zeitbezug

### ISSUE THREAD

**Issue by nichtich** [hbz/nwbib#481]
Gibt es eine Möglichkeit mit Zeitbezug zu suchen, also beispielsweise Publikationen zum 18. Jahrhundert?

**Comment by acka47** [comment]
Nein, leider nicht. Da geht die NWBib nicht über die RSWK hinaus, so dass wir keine `dct:temporal`-Informationen o.ä. angeben können.

**Comment by dr0i** [comment]
Hallo Jakob, wie Adrian schreibt, prinzipiell eher nicht. Und wenn, dann nur sehr ungenau. Die Ressourcen sind wohl nicht/kaum mit Zeitraum-Identifikatoren verschlagwortet. Es bleibt nur der Zugriff auf frei vergeben Schlagworte. Etwa so:
https://nwbib.de/search?subject=%22Geschichte%201800%22
Jedoch ist diese Suche ungenau - es werden z.B. auch Ressourcen, die _bis 1800 Jhdt_ handeln, gefunden. Und die, die z.B. eine Zeitangabe dazwischen haben, z.B. ["1779-1887"](http://lobid.org/resources/HT020105775.json), werden *nicht* gefunden.

Es scheinen jedoch [recht viele Ressourcen mit einem "Geschichtslabel"](https://nwbib.de/search?subject=%22geschichte%22) versehen. Wenn deine Frage darauf hinausläuft, ob da nicht etwa ein regex-Suche oder (wahrscheinlich besser) Metadaten verbessert/verallgemeinert und angreichert werden könnten, so scheint das nicht ganz abwegig zu sein.

**Comment by nichtich** [comment]
> Ressourcen sind wohl nicht/kaum mit Zeitraum-Identifikatoren verschlagwortet

Nicht alle Resourcen haben überhaupt einen relevanten Zeitbezug, der sich verschlagworten ließen, insofern beträfe die Eingrechnung nach Zeitbezug sowieso nur eine Teilmenge. Ich kam auf die Frage über lokale Zeitschlagwörter in der Hamburger Regionalbibliographie, die bisher nicht ausgewertet werden. Ein Mapping auf (oder Integration in) PeriodO wäre dazu sinnvoll (siehe [Idee das in Cocoda anzubieten](https://github.com/gbv/cocoda/issues/75)). Das ganze Thema ist wohl eher Idee für ein eigenes Forschungsprojekt wobei Regionalbibliographien einen passenden Anwendungsfall liefern.

**Comment by dr0i** [comment]
Heute mit @acka47 kurz versucht zu eruieren wieviele "Geschichts"-label eine Jahresangabe beinhalten. Dabei recht unscharf gefiltert und eine Stichprobe auf 1000 Ressourcen gemacht:

```curl "https://nwbib.de/search?location=&q=subject.componentList.label:\"Geschichte\"&size=1000&format=jsonld" | jq .member[].subject[].label |grep Geschichte |grep  \| | sed 's#^".*Geschichte\(.*\)"#Geschichte\1#g'```
Das bringt 1194 `componentList`-label mit "Geschichte" drin, davon haben "988" eine Zeitangabe von 1000-2999. Stichprobe könnte auch nicht ausagekräftig sein - ich glaube aber die Datenlage ist ziemlich gut. Geschätzt M.E. haben 1/4 aller Nwbib-Ressourcen SubjectHeadings mit Zeitangaben.
todo: das mal genauer über alle Ressourcen bestimmen.
